apps/server/src/shared/repo/user/user-do.repo.ts

```diff
@@ -105,8 +105,8 @@ export class UserDORepo extends BaseDORepo<UserDO, User, IUserProperties> {
 			previousExternalId: entity.previousExternalId,
 		});
 
-		if (entity.roles.isInitialized(true)) {
-			user.roleIds = entity.roles.getItems().map((role: Role) => role.id);
+		if (entity.roles.isInitialized()) {
```

Comment:
Did you check why this was marked true before?

Reply:
yes and it is not necessary for us. removing it makes the call lighter